test/integration/targets/azure_rm_virtualmachine/inventory/group_vars/all.yml

```diff
@@ -1,8 +1,21 @@
+ansible_connection: local
+ansible_python_interpreter: "{{ ansible_playbook_python }}"
```

Comment:
Keep all the inventory details in one place by putting these vars into the inventory file.

Reply:
I actually prefer to have variables in separate files rather than inventory. I'll go ahead and change it, though.

Reply:
These variables are specific to the hosts used to run the tests, rather than the tests themselves. We will soon have the ability to run tests using connection plugins other than local, which will require generating inventory for tests like this. Having the variables outside of inventory will make that more difficult to accomplish.

Reply:
Oh, that's a good case for keeping it all together.